go.mod

```diff
@@ -43,4 +43,6 @@ require (
 // replace go.step.sm/cli-utils => ../cli-utils
 // replace go.step.sm/linkedca => ../linkedca
 
+//replace go.step.sm/linkedca => ../linkedca
```

Comment:
isn't this already in the file? just above here.

Reply:
Must've been a left over. Fixed now.